backend/src/api/agent/routes.py

```
@@ -3530,6 +3541,229 @@ async def get_previous_result_for_collection(
     )
 
 
+# ============================================================================
+# Release Management Endpoints (Issue #136 - Agent Setup Wizard)
+# ============================================================================
+
+
+@router.get("/releases/active", response_model=ActiveReleaseResponse)
+async def get_active_release(
+    db: Session = Depends(get_db),
```

Ruff (0.14.14)

[warning] Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)